en/docs/monitoring-and-insights/integrate-choreo-with-moesif.md

```diff
@@ -0,0 +1,85 @@
+# Integrate Choreo with Moesif
+
+Moesif is an API analytics and monetization service that helps you grow your API products. You can integrate Choreo with Moesif and gain valuable insights into your API's usage.  While Moesif's API analytics capabilities empower you to make informed decision-making on behalf of your organization, its monetization options simplify revenue generation from your APIs, eliminating the need for complex coding. 
+
+In this guide, you will:
+
+- Generate a key in Moesif.
+- Configure Choreo to connect to Moesif.
+- Invoke an API and observe the insights on the Moesif dashboard. 
+
+## Step 1: Generate an API key in Moesif
+
+Let's generate an API key in Moesif. 
+If you're an existing Moesif user with a registered organization, proceed to [step 1.2](#step-12-access-the-api-key-in-moesif-as-an-existing-user).
+
+### Step 1.1: Generate a key in Moesif as a new user 
+
+If you are a new user, follow the steps below to register an organization and generate a key:
+
+1. Go to [https://www.moesif.com/](https://www.moesif.com/).
+2. Click **Sign Up** and select an authentication option and proceed. 
+3. In the **Get Started** page, add meaningful values to the fields and create a new organization. 
+
+    | Field                 |  Description                                            |
+    |-----------------------|---------------------------------------------------------|
+    | **Organization Name** |  The name of your company or team.                      |
+    | **Application Name**  | An application represents one project or environment.   |
```

Comment:
Since the field is **Application Name** should we rephrase the description to perhaps say "An application name to represent your project or environment."

Reply:
How about, "The name of your application that represents one project or environment" ?